invenio_assets/assets/package.json

```diff
@@ -65,7 +65,7 @@
     "webpack-dev-middleware": "^3.4.0",
     "webpack-hot-middleware": "^2.24.3",
     "webpack-livereload-plugin": "^2.2.0",
-    "webpack-manifest-plugin": "2.0.4",
+    "webpack-bundle-tracker": "^1.0.0-alpha.1",
```

Comment:
I think we should upgrade dependencies too (run `npm outdated`). In particular there is an issue that Karolina encountered with Node 13, `node-sass` really depends on some Node versions, see [here](https://github.com/sass/node-sass#supported-nodejs-versions-vary-by-release-please-consult-the-releases-page-below-is-a-quick-guide-for-minimum-support).

With Node 13, `node-sass` should be at least `4.13`.

Maybe we should add a Travis matrix with different Node versions, from LTS (12) to 14?

Reply:
Does it need a fix in this PR, or can we merge and fix afterwards (log an issue)?